src/chess/position.h

```diff
@@ -82,6 +82,7 @@ class Position {
 };
 
 enum class GameResult { UNDECIDED, WHITE_WON, DRAW, BLACK_WON };
+enum class CertaintyTrigger { NONE, TB_HIT, TWO_FOLD, TERMINAL, NORMAL };
```

Comment:
This flag is currently only for two-folds to differentiate them from three-folds. Three-folds are still terminal draws . In a more refined version (not this PR) i used this distinction (all else being equal) to:
(1) Prefer terminal draws (three-folds) if root Q is <0.00 and two-folds if root Q is > 0.00, to allow the opponent make mistakes if we feel we would otherwise be better.

Also it allows to:
(2) Use the two-fold flag to display not 0.00 if best move is a two-fold but the Q from previous (first) time the position arose or root Q. This would solve the issue of displaying 0.00 score before the opponent diverges. SF fsome time ago added something similar to suppress these scores. Its a purely cosmetic issue, but here a flag to separate two-folds from three-folds would also come in handy.

I could rename it two REPETITION, SECOND_REPETITION or add another flag for a THREE_FOLD which could be set in terminals (even though the node being terminal already indicates that its a three_fold, because two_folds do not terminate the game and are just scored differently). Any name is fine by me.....